plugins/modules/swim_workflow_manager.py

```diff
@@ -34,6 +34,16 @@
     description: Set to True to verify the Cisco Catalyst Center config after applying the playbook config.
     type: bool
     default: False
+  max_timeout:
```

Comment:
Can we say
dnac_api_task_timeout  (or) dnac_api_task_fetch_timeout (or) dnac_task_retrieval_deadline ... 

Also we can write this documentation in plugins/doc_fragments/workflow_manager_params.py  and intent_params.py?

Reply:
added.